src/Databases/AbstractNetAcuityDatabase.php

```diff
@@ -36,23 +47,26 @@ abstract class AbstractNetAcuityDatabase implements NetAcuityDatabaseInterface
     /**
      * AbstractNetAcuityDatabase constructor.
      *
-     * @param ClientInterface $client       The injected GuzzleHttp Client.
-     * @param string          $apiUserToken The Net Acuity API User Token.
+     * @param ClientInterface $client           The injected GuzzleHttp Client.
+     * @param string          $apiUserToken     The Net Acuity API User Token.
+     * @param string          $netacuityBaseUrl The base url for the netacuity webservice.
      */
     public function __construct(
         ClientInterface $client,
-        string $apiUserToken
+        string $apiUserToken,
+        string $netacuityBaseUrl
```

Comment:
I recommend making this new parameter have a default value so that this change isn't backwards-breaking.
```suggestion
        string $netacuityBaseUrl = self::DEFAULT_NETACUITY_BASE_URL
```